src/Surfnet/StepupSelfService/SelfServiceBundle/Service/RemoteVetting/Encryption/IdentityEncrypter.php

```diff
@@ -40,26 +41,18 @@ public function __construct(RemoteVettingConfiguration $configuration, IdentityW
     }
 
     /**
-     * @param IdentityDto $identity
-     * @param string $source
+     * @param AttributeListDto $identity
+     * @throws \Exception
      */
-    public function encrypt(IdentityDto $identity, $source)
+    public function encrypt(AttributeListDto $identity)
     {
-        $data = $this->constructData($identity, $source);
+        $data = json_encode($identity->jsonSerialize());
```

Comment:
I did this in order to get the test working and this was indeed quick and dirty because this should be implemented in detail when the final method was chosen.